EVGEN/AliDecayerExodus.cxx

```diff
@@ -339,11 +377,6 @@ Double_t AliDecayerExodus::Lorentz(Float_t mass, Double_t vmass, Double_t vwidth
 // using Lorentz function
 
   Double_t weight;
-
-  if( vwidth < 1e-6 ){
-    AliWarning(Form("Width = %f (Mass = %f) < 1e-6, set to 1e-6",vwidth,vmass));
-    vwidth = 1e-6;
-  }
```

Comment:
Seems like a reasonable protection, sure it should go?

Reply:
It is indeed and should not have been removed. Thanks!
Reimplemented.